common/app/model/facia.scala

```diff
@@ -21,7 +21,7 @@ object SeoData extends GuLogging {
   val editions = Edition.all.map(_.id.toLowerCase)
 
   def fromPath(path: String): SeoData =
-    path.split('/').toList match {
+    (path.split('/').toList: @unchecked) match { // split() never gives the empty list
```

Comment:
Is this always true? What about an empty path?

Reply:
I checked the following:

```
val s = ""
s.split('/')

Evaluates:
val s: String = ""
val res0: Array[String] = Array("")


val s = " "
s.split('/')

Evaluates:
val s: String = " "
val res0: Array[String] = Array(" ")
```